src/app/models/field/baseField.ts

```diff
@@ -62,6 +62,11 @@ export const BaseFieldSchema = new Schema<IFieldSchema>(
       enum: Object.values(BasicField),
       required: true,
     },
+    allowPrefill: {
```

Comment:
Add this in typings too in `src/types/field/baseField.ts`

Reply:
Actually, @liangyuanruo do we want to add a default? Or make it an optional schema key altogether?

Reply:
eh it already defaults to false; and it is optional since required is not set?

Reply:
If we don't add the `default` key-value, fields will not have the key automatically added when saving the form. So if (big if) we need to clean up we don't need to clean up many keys and just need to remove it from the single form we add it to

Reply:
I would prefer having a default for clarity but scoped only to the Textfield model instead, in order to align with the current intended functionality. Can we move `allowPrefill` there?

Reply:
ok, adjusted accordingly